capnpc-go/nodes.go

```diff
@@ -161,6 +161,10 @@ func parseAnnotations(list schema.Annotation_List) *annotations {
 	for i, n := 0, list.Len(); i < n; i++ {
 		a := list.At(i)
 		val, _ := a.Value()
+		if val.Which() != schema.Value_Which_text {
```

Comment:
Actually, it would be more correct to inline the `val.Text()` method call. `$notag` does not have a text value.